receiver/dockerstatsreceiver/metadata.yaml

```diff
@@ -42,6 +42,10 @@ resource_attributes:
     description: "The hostname of the container."
     type: string
     enabled: true
+  container.labels:
+    description: "The labels attached to the container."
+    type: map
+    enabled: true
```

Comment:
new metrics must be introduced as disabled by default, and enabled later with a breaking change.